dids/src/main/kotlin/web5/sdk/dids/DidWeb.kt

```diff
@@ -82,7 +88,18 @@ public sealed class DidWebApi(
 
   private val mapper = jacksonObjectMapper()
 
-  private val engine: HttpClientEngine = configuration.engine ?: CIO.create {}
+  private val engine: HttpClientEngine = configuration.engine ?: OkHttp.create {
+    val appCache = Cache(File("cacheDir", "okhttpcache"), 10 * 1024 * 1024)
+    val bootstrapClient = OkHttpClient.Builder().cache(appCache).build()
+
+    val dns = DnsOverHttps.Builder().client(bootstrapClient)
```

Comment:
why is this preferable? why choose 8.8.8.8 over 9.9.9.9 (quad9) or 1.1.1.1 (cloudflare)?

Reply:
This wasn't intentionally chosen. I copy pasted the setup as shown in https://github.com/square/okhttp/tree/master/okhttp-dnsoverhttps

Do you have any recommendations?

Reply:
I would recommend quad9 for its strong privacy characteristics

Reply:
Done in [9459174](https://github.com/TBD54566975/web5-kt/pull/140/commits/9459174f3c1b317af0210ef301429c45e5c2e998)